console/src/main/kotlin/app/accrescent/parcelo/console/routes/auth/GitHub.kt

```diff
@@ -6,9 +6,8 @@ import app.accrescent.parcelo.console.data.Users
 import app.accrescent.parcelo.console.data.WhitelistedGitHubUser
 import app.accrescent.parcelo.console.data.WhitelistedGitHubUsers
 import io.ktor.client.HttpClient
-import io.ktor.http.HttpMethod
-import io.ktor.http.HttpStatusCode
-import io.ktor.server.application.call
+import io.ktor.http.*
+import io.ktor.server.application.*
```

Comment:
I try to only use explicit imports. I can add this to CONTRIBUTING.md once that's created. You can update Intellij IDEA's auto-formatter settings by enabling Editor -> Code Style -> Kotlin -> "Imports" tab -> "Use single name import."